### DISCUSSION TITLE
Upstream and regular proxy

### DISCUSSION THREAD

**Discussion by gianluca-sabena** [mitmproxy/mitmproxy#6060] [Q&A]
Dear all,
I need to use mitmproxy in this use case:
* do not intercept https traffic  
* for same destination (all domain *.company-a.com, *.company-b.com ip x.y.z.n)  act as a regular proxy (mode regular proxy)
* for all other domains forward traffic to another  http proxy server (mode upstream)

Is this possible?
How can I configure mitmproxy?

Thank you!

**Comment by uesleivsouza** [comment]
> Dear all, I need to use mitmproxy in this use case:
> 
> * do not intercept https traffic
> * for same destination (all domain *.company-a.com, *.company-b.com ip x.y.z.n)  act as a regular proxy (mode regular proxy)
> * for all other domains forward traffic to another  http proxy server (mode upstream)
> 
> Is this possible? How can I configure mitmproxy?
> 
> Thank you!

Hello, Did you find how to do it? I have the same situation, need to upstream part of the traffic to another proxy and ignore specifics hosts, I tried use upstream and ignore-hosts but it seems don't work together ...
